RFS/src/main/java/org/opensearch/migrations/bulkload/version_universal/RemoteDataProvider.java

```diff
@@ -21,15 +21,15 @@ public List<Snapshot> getSnapshots() {
     }
 
     @Override
-    public List<Index> getIndicesInSnapshot(String snapshotName) {
+    public List<Index> getIndicesInSnapshot() {
```

Comment:
Really appreciate this refactor - for your consideration breakout the 'fix' for the snapshot name into its own PR so we could fast-track merging of that change and isolate it from the other work that you are doing.